website/src/pages/pricing.tsx

```diff
@@ -48,120 +72,152 @@ export default ((props: any) => (
+                                            Unlimited users
                                         </PricingPlanProperty>
                                         <PricingPlanProperty className="mt-3">
-                                            SLA with dedicated support
+                                            SLA with dedicated customer engineer<br/>and private Slack channel
                                         </PricingPlanProperty>
                                     </>
                                 }
-                                features={{
-                                    ...TEAM_FEATURES,
-                                    singleSignOn: true,
-                                    batchChanges: true,
-                                    multipleCodeHosts: true,
-                                    repositoryPermissions: true,
-                                    optimizedRepositoryUpdates: true,
-                                    privateExtensions: true,
-                                    deploymentMetricsAndMonitoring: true,
-                                    backupRestore: true,
-                                    customBranding: true,
-                                    onlineTraining: true,
-                                    customContractLegalBillingTerms: true,
-                                    unlimitedCode: true,
-                                }}
-                                beforeBatchesFragment={
-                                    <span className="text-muted pl-4 small mt-4">Available add-ons:</span>
-                                }
+                                features={ENTERPRISE_FEATURES}
+                                isFree={false}
                                 buttonLabel="Contact us"
                                 buttonClassName="btn-outline-primary"
                                 buttonHref="/contact/request-info/?form_submission_source=pricing-enterprise"
                             />
                         </div>
                     </div>
+
+                    <div className="row justify-content-center pt-md-4">
+                        <div className="col-lg-10 text-center">
+                            <blockquote className="blockquote pricing-page__quote pricing-page__quote--in-content">
+                                <p>
+                                    Sourcegraph gives us the ability to search for and refactor references to deprecated services, libraries, URL patterns, and more across our 2000+ repositories, and the confidence that we're not leaving anyone behind.
+                                </p>
+                                <footer className="blockquote-footer">Aneesh Agrawal, Software Engineer, Lyft</footer>
+                                <div className="d-flex justify-content-center my-4">
+                                    <img src="/external-logos/lyft-logo.svg" width="87px" alt="Lyft" />
```

Comment:
@jeanduplessis - can this logo link here https://about.sourcegraph.com/case-studies/lyft-monolith-to-microservices/